includes/functions.php

```diff
@@ -1854,6 +1854,11 @@ function ap_sanitize_unslash( $str, $from = false, $default = '' ) {
 		return array_map( 'sanitize_text_field', $str );
 	}
 
+	// preserving line breaks for the email notification message
```

Comment:
Hello @priard,
I think this code can be added directly to email extension.
Thanks